src/views/CoursesView.vue

```diff
@@ -14,6 +18,11 @@ const toast = useToast();
 let nameInput = ref();
 let descriptionInput = ref();
 let semesterInput = ref();
+let showCloneCourse = ref(false);
+let cloneNameInput = ref();
+let cloneDescriptionInput = ref();
+let cloneSemesterInput = ref();
+let cloneCourseId = ref();
```

Comment:
```suggestion
let cloneNewName = ref("");
let cloneNewDescription = ref("");
let cloneNewSemester = ref("");
let cloneOldCourseId = ref(-1);
```
Or are you counting on it being null?
Because otherwise, it's probably less error-prone to initialize with the empty string.
Also, renamed to improve readability.